lib/bolt/cli.rb

```diff
@@ -186,17 +182,16 @@ def load_config
     # Completes the setup process by configuring Bolt and log messages
     def finalize_setup
       Bolt::Logger.configure(config.log, config.color)
+      Bolt::Logger.disable_warnings = config.disable_warnings
```

Comment:
This seems like it should either be part of `Bolt::Logger.configure` or `disable-warnings` should be under the `log` key in config.